init.py

```diff
@@ -6,27 +6,32 @@
 """
 import sys
 from neurodamus import commands
-from neuron import h
 
 
-def main():
+def extract_arguments(args):
     """Get the options for neurodamus and launch it.
 
     We can't use positional arguments with special so we look for
     --configFile=FILE, which defaults to simulation_config.json
     """
-    first_argument_pos = 1
     config_file = "simulation_config.json"
-
-    for i, arg in enumerate(sys.argv):
-        if arg.endswith("init.py"):
-            first_argument_pos = i + 1
+    for i, arg in enumerate(args[1:]):
+        if not arg.startswith("-"):
+            raise ValueError("Positional arguments are not supported")
         elif arg.startswith("--configFile="):
             config_file = arg.split('=')[1]
-            first_argument_pos = i + 1
             break
+    args = [config_file] + [x for x in args[1:] if not x.startswith("--configFile=")]
+    return args
 
-    args = [config_file] + sys.argv[first_argument_pos:]
+
+def main():
+    try:
+        args = extract_arguments(sys.argv)
+    except ValueError:
+        print("Positional arguments are not supported by init.py. Please specify --configFile=<config file> to\n"
```

Comment:
I would prefer to let neuron `h.quit(exitcode)` to handle the proper abort in particular for MPI processes. How about `logging.error` and return a code here ?

Reply:
done

Reply:
> The only thing, Im not sure logging is configured at this point

At this point, logging just simply throws `ERROR:root:Positional arguments are not supported ....`
We also have an early call in `command.py` . 
```
    if not os.path.isfile(config_file):
        logging.error("Config file not found: %s", config_file)
        return 1
```
That seems ok, right? @ferdonline